torchrl/envs/common.py

```diff
@@ -1253,7 +1249,11 @@ def policy(td):
             tensordict = step_mdp(
                 tensordict,
                 keep_other=True,
-                exclude_action=False,
+                exclude_action=True,
```

Comment:
If we set `exclude_action=True` we may save time.
Was there a reason it was set differently?
 I might be missing something

Reply:
There was probably a reason why but I can't recall
If tests pass without it i'm open to the change